logbook-api/src/main/java/org/zalando/logbook/StructuredHttpLogFormatter.java

```diff
@@ -65,6 +65,9 @@ default Map<String, Object> prepare(final Precorrelation precorrelation, final H
         content.put("scheme", request.getScheme());
         content.put("port", preparePort(request));
 
+        if (!request.getAttributes().isEmpty())
```

Comment:
suggestion: should the attributes also be added to the `DefaultHttpLogFormatter` ?